<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="Image">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>430</height>
          </size>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Цена до скидки</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Цена после скидки</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Зачислят бонусов при покупки</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Ссылка на товар</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="BasePrice">
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="SalePrice">
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="BonusRubles">
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="Link">
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
